addons/auth_signup/controllers/main.py

```diff
@@ -58,7 +58,12 @@ def web_auth_signup(self, *args, **kw):
                 )
                 template = request.env.ref('auth_signup.mail_template_user_signup_account_created', raise_if_not_found=False)
                 if user_sudo and template:
-                    template.sudo().send_mail(user_sudo.id, force_send=True)
+                    template.sudo().with_context(
+                        email_notification_force_header=True,
+                        email_notification_force_footer=True,
+                    ).send_mail(user_sudo.id, force_send=True,
+                                email_layout_xmlid='mail.mail_notification_layout',
+                                subtitles=[_("Your Account"), user_sudo.name or ''])
```

Comment:
:socks: We can use "send_mail_batch" (plan to remove the single one), and also indent can be improved a bit :) 
```suggestion
                    ).send_mail_batch(
                        user_sudo.ids,
                        force_send=True,
                        email_layout_xmlid='mail.mail_notification_layout',
                        subtitles=[_("Your Account"), user_sudo.name or ''],
                    )
```